pkg/controller/clusterpool/collections.go

```diff
@@ -584,6 +599,252 @@ func (cds *cdCollection) Delete(c client.Client, cdName string) error {
 	return nil
 }
 
+type cdcCollection struct {
+	// Unclaimed by any cluster pool CD any not broken
+	unassigned []*hivev1.ClusterDeploymentCustomization
+	// Listed in the cluster pool inventory but not found
```

Comment:
Meaning no CDC object of that name was found on the server, right? (I don't know how to phrase this better.)

Reply:
I changed it to: "Missing CDC means listed in pool inventory but the custom resource doesn't exist in the pool namespace"